student.py

```diff
@@ -9,40 +9,43 @@ def __init__(self):
         self.students = {}
 
     def add_student(self, name, age, student_id):
-        if student_id in self.students:
+        # Error: Incorrect condition, should check if student_id in self.students
+        if student_id not in self.students:  
             print(f"Error: A student with ID {student_id} already exists.")
         else:
             self.students[student_id] = Student(name, age, student_id)
 
     def get_student(self, student_id):
-        return self.students.get(student_id, None)
+        # Error: Missing return statement
+        self.students.get(student_id, None) 
```

Comment:
The method is missing a return statement, which is necessary to return the student object.
```diff
-        self.students.get(student_id, None)
+        return self.students.get(student_id, None)
```
---

Reply:
@sparklebot hello testing 123 12:12pm

Reply:
@sparklebot hello testing 123 12:14pm

Reply:
@sparklebot hello testing 123 12:22pm